resources/tables.go

```diff
@@ -52,6 +62,14 @@ func PeopleTable() *schema.Table {
 	}
 }
 
+func PeopleProfileInfoTable() *schema.Table {
+	return &schema.Table{
+		Name:      "galaxies_people_profile_info_table",
```

Comment:
Probably too late, given the other tables use this naming pattern, but I'm not sure we need the `_table` suffix?

Reply:
Yep, think this 🚢 has sailed already, though agree the suffix feels unnecessary.